src/poprox_storage/repositories/account_interest_log.py

```diff
@@ -85,6 +86,103 @@ def fetch_topic_preferences(self, account_id: UUID) -> list[AccountInterest]:
         ]
         return results
 
+    def fetch_entity_preferences(self, account_id: UUID) -> list[dict]:
+        current_interest_tbl = self.tables["account_current_interest_view"]
+        entity_tbl = self.tables["entities"]
+
+        # Exclude topic entities that have separate preference forms
```

Comment:
We should probably be selecting or excluding by type instead of a hard-coded list